### PR TITLE
add Flipcountdown v3.0.5 with npm auto-update 

### LINKED ISSUE #7627: [Request] Add flipcountdown
https://github.com/xdan/flipcountdown

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/33121141-request-add-flipcountdown?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by nolsherry**
@pvnr0082t, please help me to review this pull request,thanks!! :grinning: 
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/xdan/flipcountdown
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/flipcountdown
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 128    
   - Count of watchers: 32
   - Count of forks: 72
 * NPM download stats (optional):
   - Downloads in the last day: 4 
   - Downloads in the last week: 9
   - Downloads in the last month: 43

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [X] The lib has notable popularity
   * [X] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [X] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [X] Has valid tags for each versions (for git auto-update)
 * [X] Auto-update setup
 * [X] Auto-update target/source is valid.
 * [X] Auto-update filemap is correct.

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [X] Separate unrelated changes into different commits.
 * [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X] Close corresponding issue in commit message
 * [X] Mention related issue(s), people in commit message, comment.

close #7627, cc @xdan

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
1. PR mixed with unrelated commit.
2. Please fix #7706 first.

File: ajax/libs/flipcountdown/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "flipcountdown",
+  "version": "3.0.5",
+  "description": "jQuery Clock,Timer or Counter plugin in retro flip-count down style. The plugin can display the time and number, including floating point.",
+  "npmName": "flipcountdown",
+  "filename": "jquery.flipcountdown.min.js",
```
Comment (pvnr0082t) [inline ajax/libs/flipcountdown/package.json]: @nolsherry Why your npm auto-update config split by filename? Does'nt you copy from [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md) or other libraries with npm autoupdate? It looks weird.

File: ajax/libs/flipcountdown/3.0.5/jquery.flipcountdown.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+.xdsoft_flipcountdown >.xdsoft_digit{
+	background-position:0px 0px;
+}
+.xdsoft_flipcountdown >.xdsoft_digit.xdsoft_separator,.xdsoft_flipcountdown >.xdsoft_digit.xdsoft_dot{
+	opacity:0.9;
+}
+.xdsoft_flipcountdown.xdsoft_size_lg >.xdsoft_digit{
+	width:53px; 
+	height:76px;
+	background-image:url(./img/digit-lg.png);
```
Comment (pvnr0082t) [inline ajax/libs/flipcountdown/3.0.5/jquery.flipcountdown.css]: @nolsherry You also need to grab ```.png``` files in ```img``` folder.

**Comment by nolsherry** [comment]
@pvnr0082t, I have added ```*.png``` files in ```img``` folder, and updated the filemap, please help me to review this pull request, thanks!

**Comment by pvnr0082t** [comment]
@nolsherry Please keep the structure. We need to put ```.png``` files into ```img``` folder.

**Comment by nolsherry** [comment]
@pvnr0082t, I have put ```*.png``` files into ```img``` folder, please help me to review this pull request, thanks!

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Pull request merged**